src/dex-ui/pages/dao/DAOProposals/Forms/DAOTokenTransferDetailsForm.tsx

```diff
@@ -13,44 +15,66 @@ export interface TokenTransferLocationState {
 }
 
 export function DAOTokenTransferDetailsForm() {
-  /*
-   * TODO: Replace Assets Input with Dropdown
-   * const { wallet } = useDexContext(({ wallet }) => ({ wallet }));
-   * const accountTokenBalancesQueryResults = useAccountTokenBalances(wallet.savedPairingData?.accountIds[0] ?? "");
-   * const { data: tokenBalances } = accountTokenBalancesQueryResults;
-   */
   const { state } = useLocation() as TokenTransferLocationState;
   const { safeAccountId, daoType, proposalType } = useOutletContext<CreateDAOProposalContext>();
   const {
+    getValues,
     setValue,
     register,
+    watch,
     formState: { errors },
   } = useFormContext<CreateDAOTokenTransferForm>();
+  watch("tokenId");
+  const accountTokenBalancesQueryResults = useAccountTokenBalances(safeAccountId ?? "");
+  const { data: tokenBalances } = accountTokenBalancesQueryResults;
+  const selectedAsset = tokenBalances?.find((asset: TokenBalance) => asset.tokenId === getValues().tokenId);
 
   if (proposalType !== DAOProposalType.TokenTransfer) {
     setValue("type", DAOProposalType.TokenTransfer);
   }
 
-  /*
-   * TODO: Replace Assets Input with Dropdown
-   * const assetDropdownOptions =
-   *   tokenBalances
-   *     ?.filter((asset: TokenBalance) => asset.symbol !== "ℏ")
-   *     ?.map((asset: TokenBalance) => {
-   *       const { symbol, tokenId } = asset;
-   *       return {
-   *         label: symbol,
-   *         value: tokenId,
-   *       };
-   *      }) ?? [];
-   */
+  const validateAmount = (value: number) => {
+    if (value <= 0) {
+      return "Amount must be greater than 0.";
+    }
+    const tokenId = getValues().tokenId;
+    if (!tokenId) {
```

Comment:
The Assets dropdown shows a balance of `10` if no asset is selected.
![Screen Shot 2023-07-24 at 6 24 25 PM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/121b8cad-ef12-4aa4-a2ee-60bbdb50b1c4)

Reply:
You are right. Made the changes are per your suggestions.